docs/team/muraddurrani.md

```diff
@@ -11,12 +11,29 @@ meetings.
 
 ## Summary of Contributions
 
-* **Code Contributed**: `to be added soon`
-* **Enhancements Implemented**: `to be added soon`
-* **Contributions to the UG**: `to be added soon`
-* **Contributions to the DG**: `to be added soon`
-* **Contributions to the team-based tasks**: `to be added soon`
-* **Review/mentoring contributions**: `to be added soon`
-* **Contributions beyond the project team**: `to be added soon`
-* **Contributions to the Developer Guide (Extracts)**: `to be added soon`
-* **Contributions to the User Guide (Extracts)**: `to be added soon` 
+* **Code Contributed**: [tP Code Dashboard Link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-02-18&tabOpen=true&tabType=authorship&zFR=false&tabAuthor=muraddurrani&tabRepo=AY2122S2-CS2103T-W12-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+* **Enhancements Implemented**:
+  * Added classes for new types of information to be associated with a person (i.e. `Company`, `JobTitle`, etc.), and integrated them into the Model component.
+  * Implemented having a unique `Id` for a `Person`, which was used for all `Meeting` related functionality.
+  * Added `Label`, which is used to label a `Person`'s contact information. In addition, implemented functionality for automatically creating default labels when none are provided by the user, and for ensuring that duplicate labels are not created in `LabelUtil`.
+  * Implemented `ContactDetailsParser`, which is responsible for parsing user input in the `ContactDetailsWindow`.
+  * Created the `ContactDetailsWindow` and its UI components (`ContactDetailsPanel`, `ParticularsCard`, etc.), which are responsible for displaying all a person's contact details, and implemented the necessary navigation logic to change screens.
+  * Created `MeetingListPanel` and `ContactMeetingsPanel` and their UI components, which are responsible for displaying all the user's meetings, and the meetings associated with a specific contact, respectively.
+  * Implemented `AddCommand` to add a new `Person` to the Address Book.
+  * Implemented `ViewCommand` to navigate to the `ContactDetailsWindow` to view a specific `Person`'s complete contact information.
+    
+
+* **Contributions to the UG**:
+  * Added documentation for the features `edit`, `delete contact`, `clear`, `find`, `list`, and `view`.
+  * Added the Overview and Interface Guide sections as part of the Introduction.
+
+
+* **Contributions to the DG**:
+  * Added the table of user stories.
+  * Added implementation details of the `view` feature.
+
+
+* **Contributions to the team-based tasks**:
+  * Documented the purpose of the user guide in the UG.
```

Comment:
```suggestion
  * Documented the purpose of the user guide in the UG.
  
  
```

Reply:
Won't this come under contribution to UG? I think you can talk about the creation of GitHub issues or similar tasks here.

Reply:
Ya, I'm not super sure about this one cause of this example:
Updating user/developer docs that are not specific to a feature e.g. documenting the target user profile

I figured it meant that if you do some kind of UG editing or add small stuff that doesn't constitute a section then you include it here.

Reply:
Would that involve linking to the issues I've created like with the PRs?

Reply:
Yes, I think you can include your first few issues for this.